runtime/lua/vim/img/_detect.lua

```diff
@@ -0,0 +1,88 @@
+local terminal = require('vim.img._terminal')
+
+local TERM_QUERY = {
+  -- Request device attributes (DA2).
+  --
+  -- It typically returns information about the terminal type and supported features.
+  --
+  -- Response format is typically something like '\033[>...;...;...c'
+  DEVICE_ATTRIBUTES = terminal.code.ESC .. '[>q',
```

Comment:
According to https://invisible-island.net/xterm/ctlseqs/ctlseqs.html this is called `XTVERSION`. The Device Attributes is `CSI c`

```suggestion
  XTVERSION = terminal.code.ESC .. '[>q',
```